### PR TITLE
Add SVG handling to piper

### PR DESCRIPTION
**Pull request by whot**
This is the other half to the libratbag commit: https://github.com/libratbag/libratbag/pull/638

It adds the SVGs from libratbag and in the second commit hooks up the SVG loading through gresources.

### CODE REVIEW

File: data/meson.build [hunk 1]
```diff
@@ -39,3 +32,60 @@ i18n.merge_file(input: appdata,
...
+svg_mapping = files('svgs/svg-lookup.ini')
+
+test_svg_files = find_program('svgs/check-svg.py')
+test('check-svg', test_svg_files,
+     args: svgs,
+     env: ['BASEDIR=@0@'.format(meson.current_source_dir())])
+
+gresource = configure_file(input: 'piper.gresource.xml.in',
+                           output: 'piper.gresource.xml',
+                           command: ['generate-piper-gresource.xml.py',
```
Comment (Hjdskes) [inline data/meson.build]: Isn't there a built-in in Meson to do something like this? I haven't checked, just wondering out loud.

Comment (whot) [reply data/meson.build]: I couldn't find one. Replacing one line is easy  enough with `configuration_data` but in this case it's beyond meson's intentionally simple capabillities.

**Comment by whot** [comment]
FTR, @bentiss really dislikes shoving the SVGs into gresources because it requires designers to run meson/ninja to test SVGs. It should be possible to just drop the SVG in place and edit the `.ini` file and off we go.

IMO that can be a follow-up commit to this one, I didn't get to this today.

**Comment by Hjdskes** [comment]
> IMO that can be a follow-up commit to this one, I didn't get to this today.

Agreed, this works for now.

**Pull request merged**